README.md

```diff
@@ -3,8 +3,17 @@ s2-geometry-javascript
 
 Porting Google's S2 Geometry Library to Javascript
 
-
 Currently contains basic support for S2Cell
 
-More details and examples to come later.
+Simple Examples
+---------------
+
+```javascript
+var cell = S2.S2Cell.FromLatLng({ lat: 40.2574448, lng: -111.7089464 }, 15);
 
+cell.getNeighbors(); // [ cellLeft, cellDown, cellRight, cellUp ]
```

Comment:
there's no specific order to the returned cells - it's likely that different places on the world have different orders. Also, in some places they're rotated up to 45 degrees (e.g. in Japan) so 'up' doesn't even make sense.

Oh - and I'm not sure about that sample location - I suggest a sample coordinate that's clearly not residential. e.g. landmarks, or in national parks.